test/jasmine/assets/create_graph_div.js

```diff
@@ -4,5 +4,11 @@ module.exports = function createGraphDiv() {
     var gd = document.createElement('div');
     gd.id = 'graph';
     document.body.appendChild(gd);
+
+    // force the graph to be at position 0,0 no matter what
+    gd.style.position = 'fixed';
+    gd.style.left = 0;
+    gd.style.top = 0;
```

Comment:
Different browsers seem to disagree on the default body margin. Before this `gd` had its origin at something like `6,6` or `8,8`, now it's a consistent `0,0`, as long as you make it with `createGraphDiv()`. I had to change a lot of mouse positions after that, but at least now they really correspond to pixels on the plot (so if you know the exact plot layout you can calculate them) and behave consistently across the different test runners.